tests/installation/start_install.pm

```diff
@@ -108,10 +108,10 @@ sub run {
         && !get_var("UPGRADE")
         && !check_var('VIDEOMODE', 'text'))
     {
-        my $counter = 10;
+        my $counter = 20;
         while ($counter--) {
             send_key $cmd{instdetails};
-            last if check_screen 'installation-details-view', 5;
+            last if check_screen 'installation-details-view', 20;
```

Comment:
Yes, however it will only cause 15 seconds longer in any case. Because if the installation is done, the counter doesn't matter any more. But 20 seconds can way better ensure check_screen than only 5 seconds.